### PR TITLE
Resolve failing tests/PHPCS

### PR DESCRIPTION
**Pull request by johnwatkins0**
This resolves some tests that began failing after the plugin was made public. The reason they started failing is mysterious to me -- not sure whether it had to do with the plugin being made public.

I updated wpcs to the latest version to account for a new error that was appearing in newer PHP Versions ([this](https://github.com/WordPress/WordPress-Coding-Standards/issues/1847)). 

In the production code, I provided a default settings array to handle tests failing due to the function that retrieves settings returning null instead of an array.

### Benefits

Passing build action.

### Verification Process

If the GH build action passes (which the latest did), this works.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my change.
- [x] All new and existing tests passed.

### CODE REVIEW

File: includes/class-publish-tweet.php [hunk 1]
```diff
@@ -258,7 +258,7 @@ public function get_upload_data_media_id( $post ) {
 
 		$media_id = $this->upload( $file );
 
-		return $media_id ?: null;
+		return $media_id ? $media_id : null;
```
Comment (johnwatkins0) [inline includes/class-publish-tweet.php]: The new WPCS made me do this.

Comment (dinhtungdu) [reply includes/class-publish-tweet.php]: Yeah, they updated the standard, see: https://make.wordpress.org/core/2019/07/12/php-coding-standards-changes/

File: includes/utils.php [hunk 2]
```diff
@@ -79,10 +79,27 @@ function maybe_autoshare( $post_id ) {
...
+		$settings = [];
+	}
+
+	$settings = wp_parse_args( $settings, $defaults );
+
+	if ( ! empty( $key ) && isset( $settings[ $key ] ) ) {
+		return $settings[ $key ];
+	}
+
+	return empty( $key ) ? $settings : '';
```
Comment (dinhtungdu) [inline includes/utils.php]: I think we can make conditions here easier to understand, for example using single conditions only:
```
	$settings = wp_parse_args( $settings, $defaults );

	if ( empty( $key ) ) {
		return $settings;
	}

	if ( isset( $settings[ $key ] ) ) {
		return $settings[ $key ];
	}

	return '';
```

Comment (johnwatkins0) [reply includes/utils.php]: Thanks, @dinhtungdu . Updated with your suggestion (verbatim 😄 ).

**Pull request merged**